docs/website/docs/glossary.md

```diff
@@ -4,14 +4,37 @@ sidebar_position: 6
 
 # Glossary
 
-## Pipeline
+## [Resource](./walkthroughs/create-a-pipeline.md)
 
-## Run
+  If the source is an API, then the resource is an endpoint in that API. If the source is a
```

Comment:
@matthauskrzykowski says that the best metaphor for source vs. resource is the spreadsheet (source) and a tab in it (resource). Each tab has a set of columns typically each with data of a given type. A spreadsheet groups many such tabs.